### DISCUSSION TITLE
Revolt Chat Bridge

### DISCUSSION THREAD

**Discussion by Aikatsui** [42wim/matterbridge#1717] [New bridges suggestions]
**Revolt** - Communication is critical. Privacy is essential
The best and first opensource alternative to discord platform

Resources:
https://revolt.chat/
https://github.com/revoltchat
https://gitlab.insrt.uk/revolt
https://developers.revolt.chat/api/
https://github.com/revoltchat/revolt.js

**Comment by EpicLPer** [comment]
Definitely second this!

**Comment by Aikatsui** [comment]
@42wim why this closed?

**Reply by 42wim** [reply]
this isn't closed, all new bridges are moved to discussions

**Reply by Pantyhose-X** [reply]
When will the revolt bridge be added, this discussion has been going on for a year, Not yet implemented

**Comment by EinfachRuwen** [comment]
Please add this!

**Comment by hisuiki** [comment]
yep !

**Comment by Pantyhose-X** [comment]
Please add

**Comment by CrocodileCroco** [comment]
Please add

**Comment by Pantyhose-X** [comment]
@42wim an API wrapper for Revolt written in Go
https://github.com/5elenay/revoltgo
https://github.com/mellowmarshe/rebel
https://github.com/janderedev/automod

**Reply by poVoq** [reply]
those repo links are broken, but this one might work:
https://github.com/ben-forster/revolt

**Comment by TomTheDragon** [comment]
This could also be useful
https://github.com/sentinelb51/revoltgo
